Hearthstone Deck Tracker/FlyoutControls/Options/OptionsSearch.xaml.cs

```diff
@@ -103,5 +104,12 @@ public UserControlWrapper(UserControl userControl, string name)
 			public UserControl UserControl { get; set; }
 			public string Name { get; set; }
 		}
-	}
+
+        private void TextBoxSearch_TextChanged(object sender, TextChangedEventArgs e)
+        {
+            if (TextBoxSearch.Text.Length < 3)
+                return;
+            ButtonSearch_OnClick(ButtonSearch, e);
```

Comment:
You can just call `UpdateSearchResult(TextBoxSearch.Text)` here, no need to go through the event method.